tests/deploy/pvc-deleter.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
If possible better to be done via python Kube client

Reply:
In current concept this feature is required when uninstalling helm release. So implementing it using python client is less relevant. Solved by adding the following JUST commands:
purge-pvc - installs PVC deleter.
purge-tests-full - deletes exist helm release (uninstall ElasticSearch) and install PVC deleter to remove all PVCs